blocks/sitemap-section-feature-block/README.md

```diff
@@ -0,0 +1,24 @@
+# Sitemap-Section
+
+Sitemaps provide search engines with metadata regarding the specific news content on a website. Using the Sitemap, bots can quickly find the news articles contained on a site.
+This Sitemap identifies the url of every section in Site Service.
+
+This feed can be used to generate links to sitemaps like
+
+`http://wwww.example.com/arc/outboundfeeds/sitemap/category/{category}/`
```

Comment:
Drop this example, just use sitemap to section

Reply:
Makes sense! Updated